### PR TITLE
feat!: migrating chart from oben01/charts to the main repository

### PR DESCRIPTION
**Pull request by oben01**

### CODE REVIEW

**Comment by manuel-rw** [comment]
Is this ready to be reviewed? If yes, you can request the review from the maintainers group

**Comment by oben01** [comment]
@manuel-rw not yet is a draft for now

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
```
Comment (manuel-rw) [inline .github/FUNDING.yml:1]: Not needed, I'd remove

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      label: values.yaml
+      description: |
+        Please copy and paste your `values.yaml` file. **Make sure to remove/replace all secrets!**
+      render: yaml
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: |
+        Please copy and paste and relevant Helm/Pod logs. This will be automatically formatted into code, so no need for backticks.
```
Comment (manuel-rw) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:37]: Maybe instruct the user how to obtain logs? Shouldn't differ to much for different clusters using kubectl, no?

File: .github/pull_request_template.md
Comment (manuel-rw) [inline .github/pull_request_template.md:1]: Do we need different requirements here?
Maybe that the template can be built? Any code quality checks that need to pass?

File: .github/renovate.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      "fileMatch": ["^charts/obico/.*\\.yaml$"],
+      "matchStrings": [
+        "tag: sha-(?<currentDigest>.+)",
+        "appVersion: sha-(?<currentDigest>.+)"
+      ],
+      "datasourceTemplate": "git-refs",
+      "depNameTemplate": "TheSpaghettiDetective/obico-server",
+      "packageNameTemplate": "https://github.com/TheSpaghettiDetective/obico-server",
+      "currentValueTemplate": "release"
+    }
```
Comment (manuel-rw) [inline .github/renovate.json:27]: For my personal understanding, why do we need this ``smart 3D printing platform``?

File: .github/renovate.json
Comment (manuel-rw) [inline .github/renovate.json:1]: Can you convert this to json5? It is supported by Renovate: https://docs.renovatebot.com/configuration-options/

File: .github/renovate.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    {
+      "matchDatasources": ["docker"],
+      "matchPackageNames": ["ghcr.io/dgtlmoon/changedetection.io"],
+      "versioning": "loose"
+    },
+    {
+      "matchDatasources": ["docker"],
+      "matchPackageNames": ["ghcr.io/juanfont/headscale"],
+      "allowedVersions": "!/^0.23.0$/"
+    }
```
Comment (manuel-rw) [inline .github/renovate.json:43]: Always document why you allow only specific versions using the JSON5 format.

File: .github/workflows/release.yaml [hunk 5]
```diff
@@ -0,0 +1,188 @@
+name: Release
```
Comment (manuel-rw) [inline .github/workflows/release.yaml:1]: Release... what? Maybe ``Release Helm chart and push``?

File: .github/workflows/renovate.yaml [hunk 6]
```diff
@@ -0,0 +1,91 @@
+name: Renovate
```
Comment (manuel-rw) [inline .github/workflows/renovate.yaml:1]: FYI, we already have renovate repo: https://github.com/homarr-labs/renovate-runner
Also, this workflow doesn't seem have much to do with Renovate, no?
Maybe rename it?

File: charts/homarr/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+apiVersion: v2
+name: homarr
+description: A Helm chart to deploy homarr for Kubernetes
+home: https://homarr-labs.github.io/charts/charts/homarr/
+type: application
+version: 0.0.1
+# renovate datasource=docker depName=ghcr.io/homarr-labs/homarr
```
Comment (manuel-rw) [inline charts/homarr/Chart.yaml:7]: Remove?

File: development/kind-with-registry.sh
Comment (manuel-rw) [inline development/kind-with-registry.sh:1]: Like your comments & documentation here. Well done 👍

**Review by manuel-rw — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md
Comment (oben01) [reply .github/pull_request_template.md:1]: I just update the contribution guide with all needed requirements + tasks

File: .github/renovate.json (hunk 3 above)
Comment (oben01) [reply .github/renovate.json:27]: I just used an old renovate file. sorry about that

File: .github/renovate.json (hunk 4 above)
Comment (oben01) [reply .github/renovate.json:43]: This rule is not needed i just remove it

File: .github/workflows/renovate.yaml (hunk 6 above)
Comment (oben01) [reply .github/workflows/renovate.yaml:1]: i just rename it to Update Chart Metadata

**Comment by oben01** [comment]
@manuel-rw I just push the RV changes please let me know if i miss something

**Review by manuel-rw — APPROVED** [review]

**Pull request merged**